src/test/java/uk/gov/hmcts/reform/hmc/validator/BeanValidatorTest.java

```diff
@@ -507,4 +510,33 @@ void whenInvalidPartyRoleIsEmpty() {
         assertTrue(validationErrors.contains(PARTY_ROLE_EMPTY));
     }
 
+    @Test
+    @Disabled("test prior to HMAN-146 make nullable change, now use shouldSucceedWhenGroupReasonIsNullForHman146()")
```

Comment:
Just wondering, should this be removed since it's not executed? Or maybe it could be amended to check for a null groupLinkType element.

Reply:
Sure. It's no longer useful so will remove.